src/wincurse.cpp

```diff
@@ -126,6 +129,36 @@ void WinDestroy()
     }
 };
 
+// creates a backbuffer to prevent flickering
+void create_backbuffer()
+{
+    if( WindowDC != NULL ) {
+        if( ReleaseDC(WindowHandle, WindowDC) == 0 ) {
+            WindowDC = 0;
```

Comment:
Setting this to 0 (why not `nullptr`, it's not of a numeric type, it's a pointer) is quite pointless. It's overridden *unconditionally* a few lines below anyway. Same with `backbuffer`.